packages/clerk-js/src/ui/common/CurrentPlanSection.tsx

```diff
@@ -0,0 +1,54 @@
+import type { BillingPlanResource } from '@clerk/types';
+
+import { Box, localizationKeys, Text, useLocalizations } from '../customizables';
+import { ProfileSection } from '../elements';
+import { mqu } from '../styledSystem';
+import { centsToUnit, getRelativeToNowDateKey } from '../utils';
+
+export const CurrentPlanSection = ({ currentPlan }: { currentPlan?: BillingPlanResource | null }) => {
+  const { t } = useLocalizations();
+
+  if (!currentPlan) {
+    return null;
+  }
+
+  return (
+    <ProfileSection.Root
+      title={localizationKeys('billing.currentPlanSection.title')}
+      id='currentPlan'
+      centered={false}
+      sx={{ [mqu.md]: { alignItems: 'flex-start' } }}
+    >
+      <ProfileSection.Item id='currentPlan'>
+        <Box>
+          <Text
+            variant='subtitle'
+            sx={t => ({ marginBottom: t.space.$2 })}
+          >
+            {currentPlan.name}
+          </Text>
+          <Text sx={t => ({ fontWeight: t.fontWeights.$semibold, fontSize: t.fontSizes.$lg })}>
+            {centsToUnit(currentPlan.priceInCents)}{' '}
```

Comment:
We should use the same function to `OrganizationPlan` as well.

Reply:
yes, lets address in a following pr